app/models/book.rb

```diff
@@ -2,6 +2,8 @@ class Book < ApplicationRecord
   belongs_to :user
   has_many :bookings
 
+  mount_uploader :picture, PhotoUploader
```

Comment:
think we need to do this through a migration, me and @jdharveng  agreed today that we would try never to make direct changes to the model again(because of the many drops and other issues), and it would be always through migration. on the lecture slides there is the slide with the migration AddPhotoToArticles...we can discuss this and make this before merging tomorrow morning.

Reply:
Picture attribute was added through a migration. To mount the uploader needs to be directly on the model as per slides and per documentation here:
https://github.com/carrierwaveuploader/carrierwave